### PR TITLE
Made changes to util file

### PR DESCRIPTION
**Pull request by darylchang123**
- Move util constants (SHUFFLE_SEED, SHUFFLE_BUFFER_SIZE, IMG_SIZE into function params as defaults
- Add helper function (init_env) to initialize environment variables and seeds
-  Reorganize utils file into three sections (initialization, dataset, model building/training)
- Change base model to take dropout and regularizers as params; use dropout 0.2 and learning rate 0.1 as defaults

These changes are backwards compatible so existing calls to util functions should continue to work.

### CODE REVIEW

**Comment by darylchang123** [comment]
@technoapurva Could you take a look at these changes? Sorry this is a little hard to read due to many changes being combined together.

To make it easier, you can look at the individual commit changes by going to [Files changed -> Changes from all commits], and selecting the changes for a particular commit instead.

File: utils/ml_utils.py [hunk 1]
```diff
@@ -200,48 +209,100 @@ def build_model(
             padding='same',
             activation='relu',
             kernel_initializer=initializer,
+            kernel_regularizer=regularizers.l1_l2(l1=l1_regularizer, l2=l2_regularizer),
+            bias_regularizer=regularizers.l1_l2(l1=l1_regularizer, l2=l2_regularizer)
         ),
         layers.MaxPooling2D(),
+        layers.Dropout(
+            rate=dropout_rate,
+            seed=seed_value
```
Comment (technoapurva) [inline utils/ml_utils.py:218]: @darylchang123 Where is this seed_value coming from?

Comment (darylchang123) [reply utils/ml_utils.py:218]: It's passed as a parameter into the function (build_model)

Comment (technoapurva) [reply utils/ml_utils.py:218]: Makes sense! Somehow I didn't see that.

**Review by technoapurva — APPROVED** [review]

**Review by technoapurva — APPROVED** [review]

**Comment by darylchang123** [comment]
Cool, thanks for reviewing!
